src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/CodeActions/CodeActionResolutionEndpoint.cs

```diff
@@ -8,11 +8,15 @@
 using System.Threading.Tasks;
 using Microsoft.AspNetCore.Razor.LanguageServer.CodeActions.Models;
 using Microsoft.Extensions.Logging;
+using Newtonsoft.Json.Linq;
+using OmniSharp.Extensions.LanguageServer.Protocol.Models;
 
 namespace Microsoft.AspNetCore.Razor.LanguageServer.CodeActions
 {
-    internal class CodeActionResolutionEndpoint : ICodeActionResolutionHandler
+    internal class CodeActionResolutionEndpoint : ICodeActionResolveHandler
     {
+        private static readonly string AssociatedServerCapability = "codeActionsResolveProvider";
```

Comment:
nit: I'd actually name this `CodeAcdtionsResolveProviderCapability` only because when seeing it below you immediately need to scroll up to see what it actually equals; whereas if it's someone related to the content you can make an assumption as to what it is without looking elsewhere.